corehq/apps/domain/templates/domain/admin/commtrack_action_table.html

```diff
@@ -23,7 +23,7 @@
             <select class="form-control" data-bind="value: type, options: $parent.action_types, optionsText: 'label', optionsValue: 'value'"></select>
           </td>
           <td>
-            <button type="button" class="btn btn-danger" data-bind="click: $parent.remove_action"><i class="icon-trash icon-white"></i> {% trans "Remove" %}</button>
+            <button type="button" class="btn btn-danger" data-bind="click: $parent.remove_action"><i class="fa fa-remove"></i> {% trans "Remove" %}</button>
```

Comment:
Elsewhere in this file, `"css: { 'error': keyword_error }"` should be `"css: { 'has-error': keyword_error }"` and `help-inline` can be removed (replace with `help-block`?).